### PR TITLE
[ENGDOCS-910] Add new audit log categories

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Added audit log events for:
 - Organization Member Added
 - Single Sign-On domain added
 - Single Sign-On domain removed
 - Single Sign-On domain verified

### Related issues (optional)
ENGDOCS-910

### CODE REVIEW

**Comment by craig-osterhout** [comment]
@BuraChuhadar PTAL. I was unable to verify "Organization Member Added" wording on staging.

**Review by BuraChuhadar — APPROVED** [review]
Thank you @craig-osterhout, LGTM.

**Comment by BuraChuhadar** [comment]
> @BuraChuhadar PTAL. I was unable to verify "Organization Member Added" wording on staging.

I pinged our team channel to get it reproduced on staging.

**Comment by BuraChuhadar** [comment]
@vrunoa fyi.

File: docker-hub/audit-log.md [hunk 1]
```diff
@@ -66,6 +67,9 @@ Refer to the following section for a list of events and their descriptions:
 | Registry Access Management registry added | Activities related to the addition of a registry |
 | Registry Access Management registry removed | Activities related to the removal of a registry |
 | Registry Access Management registry updated | Details related to the registry that was updated |
+| Single Sign-On domain added | Details of the Single Sign-On domain added to your organization |
+| Single Sign-On domain removed | Details of the Single Sign-On domain removed from your organization |
+| Single Sign-On domain verified | Details of the Single Sign-On domain verified for your organization |
```
Comment (dvdksn) [inline docker-hub/audit-log.md:72]: Let's sentence-case this?

```suggestion
| Single Sign-On domain added | Details of the single sign-on domain added to your organization |
| Single Sign-On domain removed | Details of the single sign-on domain removed from your organization |
| Single Sign-On domain verified | Details of the single sign-on domain verified for your organization |
```

**Review by dvdksn — APPROVED** [review]
🤩

**Pull request merged**